scripts/structure-check.sh

```
@@ -0,0 +1,165 @@
+# List all .md files from rfcs subfolders into an array
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)